### PR TITLE
add aws release 20.0.0

### PR DESCRIPTION
**Pull request by whites11**
### Checklist
- [ ] Roadmap issue created
- [ ] Release uses latest stable Flatcar
- [ ] Release uses latest Kubernetes patch version

### CODE REVIEW

**Comment by whites11** [comment]
cncf are failing

```
Summarizing 38 Failures:
  [FAIL] [sig-apps] Job [It] should create pods for an Indexed job with completion indexes and specified hostname [Conformance]
  test/e2e/apps/job.go:200
  [FAIL] [sig-network] Services [It] should have session affinity work for NodePort service [LinuxOnly] [Conformance]
  test/e2e/network/service.go:3807
  [FAIL] [sig-network] Services [It] should be able to change the type from NodePort to ExternalName [Conformance]
  test/e2e/network/service.go:3908
  [FAIL] [sig-api-machinery] Garbage collector [It] should orphan pods created by rc if delete options say so [Conformance]
  vendor/github.com/onsi/ginkgo/v2/internal/suite.go:605
  [FAIL] [sig-network] Services [It] should be able to create a functioning NodePort service [Conformance]
  test/e2e/network/service.go:1283
  [FAIL] [sig-apps] Job [It] should manage the lifecycle of a job [Conformance]
  test/e2e/apps/job.go:553
  [FAIL] [sig-apps] Job [It] should apply changes to a job status [Conformance]
  test/e2e/apps/job.go:472
  [FAIL] [sig-cli] Kubectl client Kubectl describe [It] should check if kubectl describe prints relevant information for rc and pods  [Conformance]
  test/e2e/framework/util.go:630
  [FAIL] [sig-api-machinery] ResourceQuota [It] should create a ResourceQuota and capture the life of a replication controller. [Conformance]
  test/e2e/apimachinery/resource_quota.go:403
  [FAIL] [sig-apps] ReplicationController [It] should test the lifecycle of a ReplicationController [Conformance]
  test/e2e/apps/rc.go:151
  [FAIL] [sig-network] Proxy version v1 [It] should proxy through a service and a pod  [Conformance]
  test/e2e/network/proxy.go:180
  [FAIL] [sig-network] Services [It] should be able to change the type from ExternalName to NodePort [Conformance]
  test/e2e/network/service.go:1467
  [FAIL] [sig-network] Service endpoints latency [It] should not be very high  [Conformance]
  vendor/github.com/onsi/ginkgo/v2/internal/suite.go:605
  [FAIL] [sig-network] HostPort [It] validates that there is no conflict between pods with same hostPort but different hostIP and protocol [LinuxOnly] [Conformance]
  vendor/github.com/onsi/ginkgo/v2/internal/suite.go:605
  [FAIL] [sig-network] Services [It] should have session affinity work for service with type clusterIP [LinuxOnly] [Conformance]
  test/e2e/network/service.go:3807
  [FAIL] [sig-network] Services [It] should be able to switch session affinity for service with type clusterIP [LinuxOnly] [Conformance]
  test/e2e/network/service.go:3807
  [FAIL] [sig-apps] ReplicationController [It] should serve a basic image on each replica with a public image  [Conformance]
  test/e2e/apps/rc.go:437
  [FAIL] [sig-network] Services [It] should be able to switch session affinity for NodePort service [LinuxOnly] [Conformance]
  test/e2e/network/service.go:3807
  [FAIL] [sig-node] Ephemeral Containers [NodeConformance] [It] will start an ephemeral container in an existing pod [Conformance]
  test/e2e/common/node/ephemeral_containers.go:73
  [FAIL] [sig-cli] Kubectl client Update Demo [It] should create and stop a replication controller  [Conformance]
  test/e2e/framework/util.go:630
  [FAIL] [sig-apps] CronJob [It] should not schedule new jobs when ForbidConcurrent [Slow] [Conformance]
  test/e2e/apps/cronjob.go:133
  [FAIL] [sig-storage] EmptyDir wrapper volumes [It] should not cause race condition when used for configmaps [Serial] [Conformance]
  test/e2e/storage/empty_dir_wrapper.go:388
  [FAIL] [sig-apps] Job [It] should run a job to completion when tasks sometimes fail and are locally restarted [Conformance]
  test/e2e/apps/job.go:263
  [FAIL] [sig-apps] Job [It] should adopt matching orphans and release non-matching pods [Conformance]
  test/e2e/apps/job.go:342
  [FAIL] [sig-cli] Kubectl client Kubectl patch [It] should add annotations for pods in rc  [Conformance]
  test/e2e/framework/util.go:630
  [FAIL] [sig-cli] Kubectl client Update Demo [It] should scale a replication controller  [Conformance]
  test/e2e/framework/util.go:630
  [FAIL] [sig-api-machinery] Garbage collector [It] should delete pods created by rc when not orphaning [Conformance]
  vendor/github.com/onsi/ginkgo/v2/internal/suite.go:605
  [FAIL] [sig-cli] Kubectl client Kubectl expose [It] should create services for rc  [Conformance]
  test/e2e/framework/util.go:630
  [FAIL] [sig-apps] ReplicationController [It] should adopt matching pods on creation [Conformance]
  test/e2e/apps/rc.go:590
  [FAIL] [sig-network] Services [It] should be able to change the type from ClusterIP to ExternalName [Conformance]
  test/e2e/network/service.go:3908
  [FAIL] [sig-apps] CronJob [It] should replace jobs when ReplaceConcurrent [Conformance]
  test/e2e/apps/cronjob.go:169
  [FAIL] [sig-network] Services [It] should be able to change the type from ExternalName to ClusterIP [Conformance]
  test/e2e/network/service.go:1429
  [FAIL] [sig-apps] ReplicationController [It] should release no longer matching pods [Conformance]
  test/e2e/apps/rc.go:619
  [FAIL] [sig-apps] ReplicationController [It] should surface a failure condition on a common issue like exceeded quota [Conformance]
  test/e2e/apps/rc.go:511
  [FAIL] [sig-apps] Job [It] should delete a job [Conformance]
  test/e2e/apps/job.go:313
  [FAIL] [sig-api-machinery] Garbage collector [It] should keep the rc around until all its pods are deleted if the deleteOptions says so [Conformance]
  vendor/github.com/onsi/ginkgo/v2/internal/suite.go:605
  [FAIL] [sig-apps] CronJob [It] should schedule multiple jobs concurrently [Conformance]
  test/e2e/apps/cronjob.go:78
  [FAIL] [sig-api-machinery] Garbage collector [It] should not delete dependents that have both valid owner and owner that's waiting for dependents to be deleted [Conformance]
  vendor/github.com/onsi/ginkgo/v2/internal/suite.go:605
```

**Comment by whites11** [comment]
This test fails 
```
{"msg":"FAILED [sig-network] HostPort validates that there is no conflict between pods with same hostPort but different hostIP and protocol [LinuxOnly] [Conformance]","completed":292,"skipped":5323,"failed":1,"failures":["[sig-network] HostPort validates that there is no conflict between pods with same hostPort but different hostIP and protocol [LinuxOnly] [Conformance]"]}
```

**Comment by whites11** [comment]
e2e tests are green, unfortuntely the above CNCF test is failing (and it seems a regression as we saw this before and it was fixed upstream)

**Comment by whites11** [comment]
also upgrade tests pass fine.

**Comment by mcharriere** [comment]
Could you please bump `k8s-dns-node-cache-app` to `v2.6.1`? 
I reverted the `force_tcp` option for the `.` zone as it was having some impact on the AWS Resolver under heavy load.

It'd be nice to have it here for v20. thank  you.

**Comment by T-Kukawka** [comment]
Ready for release - waiting for reviews

File: aws/v20.0.0/announcement.md [hunk 1]
```diff
@@ -0,0 +1 @@
+**Workload cluster release v20.0.0 for AWS is available**. The new major version is the first release supporting Kubernetes `v1.25`. Please remember that with Kubernetes `v1.25` the Pod Security Policies (PSP) are removed from the API, hence customer workloads should also be adjusted *BEFORE* the upgrade. From now on the Pod Security Standards (PSS) are the default for securing the clusters. This release will also be used as a base for Giant Swarm `Vintage` to Giant Swarm adoption of `Cluster API for AWS` (CAPA) migration. Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v20.0.0/).
```
Comment (stone-z) [inline aws/v20.0.0/announcement.md:1]:
```suggestion
**Workload cluster release v20.0.0 for AWS is available**. The new major version is the first release supporting Kubernetes `v1.25`. Please remember that with Kubernetes `v1.25` the Pod Security Policies (PSP) are removed from the API, so all workloads must have their PSPs removed *BEFORE* starting the upgrade. From now, on the Pod Security Standards (PSS) are the default for securing the clusters. This release will also be used as a base for Giant Swarm `Vintage` to Giant Swarm adoption of `Cluster API for AWS` (CAPA) migration. Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v20.0.0/).
```

File: aws/v20.0.0/release.yaml [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    version: 0.0.1
+  - catalog: giantswarm
+    dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    - coredns
+    - prometheus-operator-crd
+    - vertical-pod-autoscaler-crd
+    name: security-bundle
+    version: 1.6.1
```
Comment (stone-z) [inline aws/v20.0.0/release.yaml:153]:
```suggestion
    version: 1.6.2
```

File: aws/v20.0.0/README.md [hunk 3]
```diff
@@ -0,0 +1,456 @@
...
+- Update to exception-recommender (app) to v0.1.0.
+- Update to falco (app) to v0.8.0.
+- Update to kyverno-policy-operator (app) version v0.0.7.
+- Update to kyverno (app) version v0.17.2.
+- Update to starboard-exporter (app) version v0.7.8.
+- Update to trivy-operator (app) to v0.7.0.
+- Update to trivy (app) to v0.10.0.
+- Update to kyverno (app) to v0.17.5.
+- Update to exception-recommender (app) to v0.1.1.
+- Update to trivy-operator (app) to v0.7.2.
```
Comment (stone-z) [inline aws/v20.0.0/README.md:103]:
```suggestion
### security-bundle [1.6.2](https://github.com/giantswarm/security-bundle/releases/tag/v1.6.2)

#### Changed
- Update to exception-recommender (app) to v0.1.1.
- Update to falco (app) to v0.8.0.
- Update to kyverno-policy-operator (app) version v0.0.7.
- Update to kyverno (app) version v0.17.6.
- Update to starboard-exporter (app) version v0.7.8.
- Update to trivy-operator (app) to v0.7.2.
- Update to trivy (app) to v0.10.0.
- Update to kyverno (app) to v0.17.5.
```

File: aws/v20.0.0/README.md [hunk 4]
```diff
@@ -0,0 +1,456 @@
+# :zap: Giant Swarm Release v20.0.0 for AWS :zap:
+
+The new major version is the first release supporting Kubernetes 1.25. 
+
+Please remember that the Pod Security Policies (PSP) are no longer supported in this version and they have to be removed *BEFORE* the upgrade. From now on the Pod Security Standards (PSS) are the default for securing the clusters. Our docs offer additional information about [Pod Security Standards](https://docs.giantswarm.io/advanced/security-policy-enforcement/) as well as a [PSS migration guide](https://docs.giantswarm.io/advanced/security/security-policy-enforcement/cluster-admin-guide/) for cluster administrators.
+
+This release will also be used as a base for `Giant Swarm Vintage` to Giant Swarm adoption of `Cluster API for AWS` (CAPA) migration. The migration will be scheduled as any other upgrade. We will be reaching out separately to each customer to present the plan as well as the CAPA benefits.
```
Comment (stone-z) [inline aws/v20.0.0/README.md:7]:
```suggestion
This release will also be used as a base for the migration from `Giant Swarm Vintage` to `Cluster API for AWS` (CAPA). The migration will be scheduled as any other upgrade. We will be reaching out separately to each customer to present the plan as well as the CAPA benefits.
```

File: aws/v20.0.0/README.md [hunk 5]
```diff
@@ -0,0 +1,456 @@
...
+
+> **WARNING:** After upgrading to `19.3.0`, it is highly advised to begin removal of all PSPs from the cluster. Kubernetes `v1.25` removes the Pod Security Policy resource from the API, meaning workloads (like Helm charts) which still contain PSPs will fail to install after the upcoming Giant Swarm `v20` release.
```
Comment (stone-z) [inline aws/v20.0.0/README.md:9]:
```suggestion
> **WARNING:** After upgrading to `19.3.0`, it is highly advised to begin removal of all customer-managed PSPs from the cluster. Kubernetes `v1.25` removes the Pod Security Policy resource from the API, meaning workloads (like Helm charts) which still contain PSPs will fail to install after the Giant Swarm `v20` upgrade.
```

File: aws/v20.0.0/README.md [hunk 6]
```diff
@@ -0,0 +1,456 @@
...
+
+> **WARNING:** `Observability-bundle` will be upgraded to `v1.1.0` and it contains breaking changes by simplify configuration for the bundled apps. Please check our [upgrade guide](https://github.com/giantswarm/observability-bundle/blob/main/docs/upgrade.md) or reach out to your Account Engineer for more details.
```
Comment (stone-z) [inline aws/v20.0.0/README.md:11]:
```suggestion
> **WARNING:** `observability-bundle` will be upgraded to `v1.1.0`, which contains breaking changes to the configuration for the bundled apps. Please check our [upgrade guide](https://github.com/giantswarm/observability-bundle/blob/main/docs/upgrade.md) or reach out to your Account Engineer for more details.
```

**Review by stone-z — COMMENTED** [review]
Can we bump to the latest security-bundle? There are some good bug fixes in there

File: aws/v20.0.0/announcement.md [hunk 7]
```diff
@@ -0,0 +1 @@
+**Workload cluster release v20.0.0 for AWS is available**. The new major version is the first release supporting Kubernetes `v1.25`. Please remember that with Kubernetes `v1.25` the Pod Security Policies (PSP) are removed from the API, so all workloads must have their PSPs removed *BEFORE* starting the upgrade. From now, on the Pod Security Standards (PSS) are the default for securing the clusters. This release will also be used as a base for Giant Swarm `Vintage` to Giant Swarm adoption of `Cluster API for AWS` (CAPA) migration. Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v20.0.0/).
```
Comment (stone-z) [inline aws/v20.0.0/announcement.md:1]: Sorry, I forgot to change this in both places
```suggestion
**Workload cluster release v20.0.0 for AWS is available**. The new major version is the first release supporting Kubernetes `v1.25`. Please remember that with Kubernetes `v1.25` the Pod Security Policies (PSP) are removed from the API, so all workloads must have their PSPs removed *BEFORE* starting the upgrade. From now, on the Pod Security Standards (PSS) are the default for securing the clusters. This release will also be used as the base for the migration from `Giant Swarm Vintage` to `Cluster API for AWS` (CAPA). Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v20.0.0/).
```

**Review by stone-z — APPROVED** [review]
LGTM
(I forgot to align the readme with the announcement)

File: aws/v20.0.0/README.md [hunk 8]
```diff
@@ -8,7 +8,7 @@ This release will also be used as a base for `Giant Swarm Vintage` to Giant Swar
 
 > **WARNING:** After upgrading to `19.3.0`, it is highly advised to begin removal of all PSPs from the cluster. Kubernetes `v1.25` removes the Pod Security Policy resource from the API, meaning workloads (like Helm charts) which still contain PSPs will fail to install after the upcoming Giant Swarm `v20` release.
 
-> **WARNING:** `Observability-bundle` will be upgraded to `v1.1.0` and it contains breaking changes by simplify configuration for the bundled apps. Please check our [upgrade guide](https://github.com/giantswarm/observability-bundle/blob/main/docs/upgrade.md) or reach out to your Account Engineer for more details.
+> **WARNING:** `observability-bundle` will be upgraded to `v1.1.0`, which contains breaking changes to the configuration for the bundled apps. Please check our [upgrade guide](https://github.com/giantswarm/observability-bundle/blob/main/docs/upgrade.md) or reach out to your Account Engineer for more details.
```
Comment (QuentinBisson) [inline aws/v20.0.0/README.md:11]:
```suggestion
> **WARNING:** `observability-bundle` will be upgraded to `v1.1.0`, which contains breaking changes to the configuration for the bundled apps to simplify configuration, especially for GitOps. Please check our [upgrade guide](https://github.com/giantswarm/observability-bundle/blob/main/docs/upgrade.md) or reach out to your Account Engineer for more details.
```

Comment (QuentinBisson) [reply aws/v20.0.0/README.md:11]: @T-Kukawka do you think we could bump the observability bundle to 1.2.4?

Comment (T-Kukawka) [reply aws/v20.0.0/README.md:11]: if we are fine that it does not break anything, maybe worth to test real quick? then yes.

Comment (QuentinBisson) [reply aws/v20.0.0/README.md:11]: It should not but can we do a test release quick? :D

Comment (T-Kukawka) [reply aws/v20.0.0/README.md:11]: @QuentinBisson u can just take that release.yaml, apply it on the Test MC or on `garfish` named e.g. 20.0.1, adjust your app in the yaml and deploy the cluster with that release basically - we do not cut test releases

Comment (QuentinBisson) [reply aws/v20.0.0/README.md:11]: sure I'm aware. I'll test tomorrow because I'm supposed to be off today :D

Comment (T-Kukawka) [reply aws/v20.0.0/README.md:11]: sure 👍 no worries

**Review by mcharriere — APPROVED** [review]

File: aws/v20.0.0/README.md (hunk 8 above)
Comment (QuentinBisson) [reply aws/v20.0.0/README.md:11]: Marie ran a test and it worked like a charm with version 1.3.0 even :D

Comment (QuentinBisson) [reply aws/v20.0.0/README.md:11]: https://github.com/giantswarm/releases/pull/1217

Comment (T-Kukawka) [reply aws/v20.0.0/README.md:11]: approved

**Review by QuentinBisson — APPROVED** [review]

**Review by T-Kukawka — APPROVED** [review]

**Review by T-Kukawka — APPROVED** [review]

**Pull request merged**